### ISSUE TITLE
getpass import is missing

### ISSUE THREAD

**Issue by byxor** [414owen/Github-Work-Visualiser#2]
```
$ python3 ./github.py 
# Login to Github
Enter your Github username [byxor]: byxor
Traceback (most recent call last):
  File "./github.py", line 218, in <module>
    start()
  File "/home/brandon/.local/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/brandon/.local/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/brandon/.local/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/brandon/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "./github.py", line 211, in start
    login()
  File "./github.py", line 114, in login
    password = getpass("Enter your Github password: ")
NameError: name 'getpass' is not defined

```

I fixed this by adding `from getpass import getpass` to the top of the file.

May I make a PR to close this? I'd like more open source street cred.

**Comment by 414owen** [comment]
Of course! I'll assign it to you now :)

**Comment by byxor** [comment]
Very cool project by the way.

**Comment by 414owen** [comment]
Why thank you!

I'd like to say I made it on a whim, but it's actually part of an assignment. I'll add more features when I get a chance, though.

Definitely needs cli switches to specify animation rates, target user, and whether to progress continuously based on time passing or based on when commits happened, if you're interested, feel free to add switches for them. The code's a bit of a mess at the moment, I'm afraid...

I also made it all from a hotel room in Hong Kong...
